docs/understanding-tracking-design/versioning-your-data-structures/cloud/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Versioning Data Structures in BDP Cloud"
+date: "2023-03-01"
+sidebar_position: 10
+---
+
+# Versioning in BDP Cloud
+
+If you are a BDP Cloud customer, you will have used our Data Structure Builder to create your custom Data Structure for your Events and Entities.
+
+![](../../managing-data-structures-with-data-structures-builder/images/data-structures-2.png)
+
+**Breaking and non-breaking changes**
+
+At the point of publishing a schema, there are two options for versioning your data structure.
+- **Non-breaking** - a non-breaking change is backward compatible with historical data and your loader.
+- **Breaking** - a breaking change is not backwards compatible with historical data and/or your loader.
+
+The Data Structure builder will automatically select how to version up your data structure depending on the changes you have just made.
```

Comment:
Thinking as a user: “this is good to know... but what does this mean?” Can we explain that non-breaking changes get a minor version bump and major changes get a major version bump? And then what the implications are in the warehouse? There are a lot of assumptions here that we are glossing over and that would be completely unfamiliar to new users, even more so to the intended audience of BDP Cloud.

Reply:
We have a video coming soon where we're showing all that. We are just not quite ready yet to publish it.